src/state/store/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore } from 'redux';
+import rootReducer from '../reducers/rootReducers';
+import initialState from './initialState';
+
+const store = createStore(
+  rootReducer,
+  initialState,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
why do you need this?

Reply:
This is for a chrome dev tool extension. Pretty nice if i say it myself.
Check it out here: https://github.com/zalmoxisus/redux-devtools-extension

Reply:
Cool!